cmd/bonanza_browser/browser_service.go

```diff
@@ -2209,6 +2215,12 @@ func (d *messageJSONRenderer) renderMessage(m model_core.Message[protoreflect.Me
 }
 
 func (d *messageJSONRenderer) renderMessageCommon(m model_core.Message[protoreflect.Message, object.LocalReference], fields map[string][]g.Node) []g.Node {
+	if d.customRenderer != nil {
```

Comment:
What if instead of having this `if ... != nil {` check, we ensure that we **always** set `d.customRenderer? In the plain JSON (non-pretty-print) case, we set it to a plain `func(...) { return nil }`.

https://en.wikipedia.org/wiki/Null_object_pattern

Reply:
Done